src/sdk/resources/auto_detect_compilers.xrc

```diff
@@ -22,16 +22,39 @@
 					<growablerows>0</growablerows>
 					<object class="sizeritem">
 						<object class="wxListCtrl" name="lcCompilers">
-							<style>wxLC_REPORT|wxLC_SINGLE_SEL</style>
+							<style>wxLC_REPORT|wxLC_SINGLE_SEL|wxVSCROLL|wxHSCROLL</style>
 						</object>
 						<flag>wxEXPAND</flag>
-						<minsize>460,240</minsize>
 					</object>
 					<object class="sizeritem">
-						<object class="wxButton" name="btnDefault">
-							<label>Set as default</label>
+						<object class="wxGridSizer">
+							<cols>1</cols>
+							<rows>3</rows>
+							<object class="sizeritem">
+								<object class="wxButton" name="btnSetDefaultCompiler">
+									<label>Set as default</label>
+									<pos>-1,-1</pos>
+									<size>-1,-1</size>
+								</object>
+								<flag>wxALIGN_CENTER_HORIZONTAL|wxALIGN_CENTER_VERTICAL</flag>
+								<border>5</border>
+							</object>
+							<object class="sizeritem">
+								<object class="wxRadioBox" name="rbCompilerShowOptions">
```

Comment:
I've requested for this to be removed. Why are you refusing to do this?

Reply:
See the original post above for reason.